python/pyspark/sql/connect/client.py

```diff
@@ -299,7 +300,11 @@ def userAgent(self) -> str:
             raise SparkConnectException(
                 f"'user_agent' parameter should not exceed 2048 characters, found {len} characters."
             )
-        return user_agent
+        return " ".join([
+            user_agent,
```

Comment:
if we put the other two values first then we have "guarantee" on the os and python version as the first two fields. 

In addition, we should put the spark version here as well -> `from pyspark.version import __version__`

Reply:
Don't think it's wise to depend on the order of fields.

Reply:
> In addition, we should put the spark version here as well 

Done